### ISSUE TITLE
Convert TLS 1.2 to TLS 1.3 from client > proxychains > proxy

### ISSUE THREAD

**Issue by DesweR** [shunf4/proxychains-windows#49]
I have a client that only supports TLS 1.2 and old cipher suites.

I need to get a client to open modern websites.

Is it possible for proxychains to connect to my local proxy via TLS 1.3 and modern cipher suites?

**Comment by shunf4** [comment]
No. Proxychains only intercepts network connections and does SOCKS5 handshakes, and that is all.

To achieve what you said requires chaging of TLS handshake process, which is the *payload* of the TCP connection. See [mitmproxy](https://mitmproxy.org) or my modified [tcpproxy](https://github.com/shunf4/tcpproxy-shunf4-mod), but I won't guide you how to use them.
